HandheldCompanion/Controllers/NeptuneController.cs

```diff
@@ -86,22 +86,20 @@ public NeptuneController(PnPDetails details)
             DrawControls();
             RefreshControls();
 
+            List<ButtonFlags> ExtraSteamDeckButtons = new();
+            ExtraSteamDeckButtons.AddRange(new List<ButtonFlags>() { ButtonFlags.OEM1, ButtonFlags.VolumeUp, ButtonFlags.VolumeDown });
```

Comment:
I'd move VolumeUp and VolumeDown to IController VirtualButtons list. (OEM1 is already there).